newIDE/app/src/UI/Theme/DarkTheme/EventsSheet.css

```diff
@@ -26,7 +26,11 @@
  */
 .gd-events-sheet-dark-theme .rst__moveHandle {
   background: #494949 !important;
-  border: 1px solid #494949 !important;
+  border: none;
```

Comment:
Why the change the border again? Forgot what's the purpose

Reply:
It's a remnant of when I split the PRs.

But i see that on darktheme, so i've deleted the border. and it need `!important;` css rule.
![image](https://user-images.githubusercontent.com/1670670/70379903-a32ec800-1933-11ea-9f6a-845554b6eae6.png)

btw it's alredy done in #1309 , not yet merged :/

Reply:
Alright I see, this is because of this rule: 

https://github.com/4ian/GDevelop/blob/9763d6523c797b6d721b25d444156a16d89de28d/newIDE/app/src/EventsSheet/EventsTree/style.css#L12-L15

Background color and border are set here. I think we should instead:
* Remove the border in this common rule.
* Probably remove also the background in this common rule, so that we can (probably) remove the !important that is already there on the background in the dark/default theme css files.

Reply:
Note that if you want to do this in the other PR, you can remove here the changed lines.